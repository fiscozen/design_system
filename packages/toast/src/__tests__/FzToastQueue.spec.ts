import { describe, it, vi } from "vitest";
import { mount } from "@vue/test-utils";
import { FzToastQueue } from "..";

describe.concurrent("FzToastQueue", () => {
  it("matches snaphost", async ({ expect }) => {
    const wrapper = mount(FzToastQueue);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
