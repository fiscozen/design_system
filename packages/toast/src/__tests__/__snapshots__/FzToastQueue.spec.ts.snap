// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FzToastQueue > matches snaphost 1`] = `
"<div class="hoverable group">
  <div class="flex flex-col relative">
    <transition-group-stub name="list" appear="false" persisted="false" css="true"></transition-group-stub>
  </div>
  <div class="flex flex-col"></div>
</div>"
`;
